components/stacks-rpc-client/src/rpc_client.rs

```diff
@@ -67,7 +67,33 @@ pub struct PoxInfo {
 }
 
 impl PoxInfo {
-    pub fn default() -> PoxInfo {
+    pub fn mainnet_default() -> PoxInfo {
+        PoxInfo {
+            contract_id: "SP000000000000000000002Q6VF78.pox-2".into(),
+            pox_activation_threshold_ustx: 0,
+            first_burnchain_block_height: 666050,
+            prepare_phase_block_length: 100,
+            reward_phase_block_length: 2000,
+            reward_slots: 4000,
+            total_liquid_supply_ustx: 1368787887756275,
+            ..Default::default()
+        }
+    }
+
+    pub fn testnet_default() -> PoxInfo {
+        PoxInfo {
+            contract_id: "ST000000000000000000002AMW42H.pox-2".into(),
```

Comment:
Why not pox-3? cc @obycode @lgalabru

Reply:
I don't think these defaults are actually used anywhere. I think they can just be deleted so we don't need to keep them in sync. I'll check and then add a commit if that's right.

Reply:
The @obycode

Reply:
Maybe we could have @janniks cherrypicking @MicaiahReid's fix, rebase and force push in this branch?

Reply:
Oh, I missed your earlier comment. Yeah, feel free to back out or overwrite my commit that deleted this.